source/js/utils.js

```diff
@@ -275,7 +275,10 @@ NexT.utils = {
           targets  : document.scrollingElement,
           duration : 500,
           easing   : 'linear',
-          scrollTop: offset + 10
+          scrollTop: offset + 10,
+          complete : () => {
+            history.pushState(null, null, element.href);
```

Comment:
`null` isn't valid in the second parameter (not documented anywhere). Passing the empty string or `document.title` would be better. According to MDN, the empty string would be safe enough ([History.pushState() | MDN](https://developer.mozilla.org/en-US/docs/Web/API/History/pushState)).

Reply:
I decided to use `document.title` because it is more consistent with the default behavior